templates/maya/render-arnold-windows/job.template.json

```diff
@@ -142,7 +134,7 @@
                             "elevationLevel": "admin"
                         }
                     },
-                    "commandLine": "mkdir \"%AZ_BATCH_TASK_WORKING_DIR%\\thumbs\" & subst X: %AZ_BATCH_JOB_PREP_WORKING_DIR% && cmd.exe /v /c \"[variables('MayaVersions')[parameters('mayaVersion')].environmentValue] -renderer arnold -ai:lve 2 [parameters('additionalFlags')] -proj \"%AZ_BATCH_JOB_PREP_WORKING_DIR%\" -rd \"%AZ_BATCH_TASK_WORKING_DIR%\\images\" -s {0} -e {0} \"X:\\[parameters('sceneFile')]\"\" & subst X: /D",
+                    "commandLine": "mkdir \"%AZ_BATCH_TASK_WORKING_DIR%\\thumbs\" & set &  echo %AZ_BATCH_ACCOUNT_URL%>url.txt&echo %AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN%>token.txt&powershell -command \"$characters=Get-ChildItem -Path Env:\\AZ_BATCH_ACCOUNT_URL;$token=Get-ChildItem -Path Env:\\AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN;write-host 'url-length' $characters.value.length 'tokenlength' $token.value.length\"&subst X: %AZ_BATCH_JOB_PREP_WORKING_DIR% && cmd.exe /v /c \"[variables('MayaVersions')[parameters('mayaVersion')].environmentValue] -renderer arnold -ai:lve 2 [parameters('additionalFlags')] -proj \"%AZ_BATCH_JOB_PREP_WORKING_DIR%\" -rd \"%AZ_BATCH_TASK_WORKING_DIR%\\images\" -s {0} -e {0} \"X:\\[parameters('sceneFile')]\"\" & subst X: /D",
```

Comment:
this looks like logging for debugging purposes? Suggest removing

Reply:
removed